Example/Tests/CryptoTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CryptoTests.swift
+//  uPortSDK_Tests
+//
+//  Created by Aldi Gjoka on 12/5/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
```

Comment:
Please update Organization name `CocoaPods` to `ConsenSys`: Select uPortSDK in Project Navigator (Xcode top-left), then in the File Inspector (Xcode right side). This is used for new files, so you'll have to manually correct here of course.